packages/explat/src/index.ts

```diff
@@ -20,6 +20,8 @@ declare global {
 	}
 }
 
+initializeAnonId().catch( ( e ) => logError( { message: e.message } ) );
```

Comment:
This should be moved to run after tracks is enabled.

Reply:
Do you mean a check for Tracks enabled?

```js
if ( ! window.wcTracks.isEnabled ) {
	initializeAnonId().catch( ( e ) => logError( { message: e.message } ) );
}
```

Reply:
Like after a user clicks the enable tracking button, I think that is here:

https://github.com/woocommerce/woocommerce-admin/blob/ef6aa388c40abffecb512e1af49c915ebe1449a6/client/profile-wizard/steps/usage-modal.js#L67-L74

and also here:

https://github.com/woocommerce/woocommerce-admin/blob/cda3b39d082624385a885132b2879db5d3685848/client/wp-admin-scripts/beta-features-tracking-modal/container.js#L22

I think within the `.enable()` callback is best.

Reply:
Ah I see. Good call. I think a combination of what I suggested plus in the callbacks is best